lua/wikis/brawlstars/Info.lua

```diff
@@ -24,6 +24,19 @@ return {
 				lightMode = 'Brawl Stars Default allmode.png',
 			},
 		},
+		chinesebs = {
+			abbreviation = 'CN BS',
+			name = '荒野乱斗',
```

Comment:
IIRC there was some component that required table key to match lowercased name, however i fail to remember which one it was and whether this is important here.

Reply:
fwiw if the input is the same as the name it would still work (same lower function used for input and buikding the name to identifier table)
just if capitalization in the input is different the string.lower might not lower stuff properly due to it being chinese